backend/src/components/secureExchange.js

```diff
@@ -250,6 +250,12 @@ async function instituteSelection(req, res) {
   }
 }
 
+async function clearActiveSession(req) {
```

Comment:
Please add `req.session.activeInstituteTitle` as well. We might want to wrap these in a parent object in the session (req.session.activeInstitute).